apps/client/src/pages/home/Home.page.tsx

```diff
@@ -17,23 +16,34 @@ import {
 } from '../../components';
 
 export function Home(): ReactElement {
-  const { id } = useParams();
+  const { roomId } = useParams();
+  // disconnect any socket connection if user navigates to home page.
+  endSocket();
```

Comment:
This was changed from an upcoming PR #73  66ea107312d0f51fe1fb478601b601bc0b570d45